index.html

```diff
@@ -123,267 +123,277 @@ <h1>Чем мы занимаемся?</h1>
+                        <div class="program__week-number">
+                            Неделя №7
                         </div>
-                        <div class = "teacher_name">
-                            Дмитрий Иванов
+                        <div class="program__week-description">
+                            Красивая часть курса,<br> которая помогает в успехе
                         </div>
-                        <div class="specialist_in">
-                            Специалист по видеокартам
+                    </div>
+                </div>
+                <div class="program__right-program-info program__right-program-info_position-absolute">
+                    <div class="program__right-week-info">
+                        <div class="program__week-number">
+                            Неделя №2
                         </div>
-                        <button class = "biography_btn">
-                            Биография
-                        </button>
-                    </li>
-                </ul>
+                        <div class="program__week-description">
+                            Красивая часть курса,<br> которая помогает в успехе
+                        </div>
+                        <div class="program__week-number">
+                            Неделя №4
+                        </div>
+                        <div class="program__week-description">
+                            Красивая часть курса,<br> которая помогает в успехе
+                        </div>
+                        <div class="program__week-number">
+                            Неделя №6
+                        </div>
+                        <div class="program__week-description">
+                            Красивая часть курса,<br> которая помогает в успехе
+                        </div>
+                    </div>
+                </div>
+                <div class = "program__main-line"></div>
+                <div class="program__left-lines">
+                    <div class = "program__line"></div>
+                    <div class = "program__line"></div>
+                    <div class = "program__line"></div>
+                    <div class = "program__line"></div>
+                </div>
+                <div class="program__right-lines">
+                    <div class = "program__line"></div>
+                    <div class = "program__line"></div>
+                    <div class = "program__line"></div>
+                </div>
+            </div>
+        </div>
+        <div class="wrapper">
+            <div class="teachers">
+                <div class="teachers__logo">
+                    <img src="img/Ellipse%205%20(Stroke).png">
+                </div>
+                <div class="teachers__header">
+                    Ваши преподаватели
+                </div>
+                <div class="teachers__list teachers__list_position-absolute teachers__list_top teachers__list_right">
```

Comment:
Rename `teachers__list_top` in `teachers__list_top-200px` and you didn't describe `.teachers__list_right`.